cluster/addons/iso-client/example-cpu-pinning.yml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 metadata:
   annotations:
     pod.alpha.kubernetes.io/isolation-api: '{
-        "cpu-pinning": "0-1"
+        "core-affinity": "0-1"
```

Comment:
Actually I was discussing with @PiotrProkop offline that these specific CPU IDs need to go away sooner rather than later in favor of something more intent based. Otherwise this will be a big surface area of criticism unrelated to the actual mechanism we are trying to enable.